caesars-palace/blocks/slider/slider.js

```diff
@@ -129,6 +246,7 @@ export default function decorate(block) {
   };
 
   mediaChangeHandler();
+  tabletMediaQuery.addEventListener('change', mediaChangeHandler);
```

Comment:
fyi - I think all those event listeners can be made passive, actually (improves page performance).